### ISSUE TITLE
Config dialog: options are active while they should not

### ISSUE THREAD

**Issue by tezeb** [IJHack/QtPass#274]
Both master and 1.1.6.
When entering configuration dialog and option "No clipboard" is set, it's possible to set "Hide content" and "Hide password". Both shall probably be deactivated as is done when changing "Clipboard behaviour" to "No clipboard"(it works fine first time, but not on reentering config dialog).
It's also not clear what does "Hide password" do when "Hide content" is also checked. If I understand correctly, "Hide content" is superior, so "Hide password" shall be deactivated(or checked & deactivated), shouldn't it?

**Comment by annejan** [comment]
yes . . or rather just a 3 option radio button setup or a dropdown with an enum?

**Comment by annejan** [comment]
hide => none, password, all

**Comment by tezeb** [comment]
Sounds like a good idea :)
